test-cases/longevity/longevity-aws-custom-d2-workload1-5dcs.yaml

```diff
@@ -0,0 +1,270 @@
+test_duration: 1440
+n_monitor_nodes: 1
+n_db_nodes: '12 12 12 12 12'
+n_loaders: '5 1 1 1 1'
+
+# TODO: increase node type when spin up of a big cluster becomes fast enough.
+instance_type_db: 'i3en.large'
+instance_type_loader: 'c6i.2xlarge'
+
+user_prefix: 'long-custom-d2-wrkld1-big'
+enterprise_disable_kms: true
+use_preinstalled_scylla: true
+
+endpoint_snitch: GossipingPropertyFileSnitch
+internode_compression: 'all'
+client_encrypt: true
+append_scylla_yaml:
+  consistent_cluster_management: false
+  compaction_static_shares: 100
+  compaction_enforce_min_threshold: true
+  compaction_throughput_mb_per_sec: 0
+  # NOTE: workaround for the https://github.com/scylladb/scylladb/issues/19131
+  enable_repair_based_node_ops: true
+  allowed_repair_based_node_ops: "replace,removenode,rebuild,decommission"
+
+nemesis_class_name: 'SisyphusMonkey'
+nemesis_seed: '029'
+nemesis_interval: 5
+nemesis_during_prepare: true
+
+round_robin: true
+# TODO: update the stress commands to have workable "rate" values when
+#       setting up 60 DB nodes cluster becomes feasible in the test environment.
+#       It must be done proportionally.
+#       See: https://github.com/scylladb/scylladb/issues/19131
+prepare_write_cmd:
+  # NOTE: --duration in these commands is number of rows that will be written.
+  #       Time gets specified with 's', 'm' or 'h' letters.
+
+  # NOTE: [region-1, loaders 1-5] first group of loaders from a single/first region write to all per-region main tables
+  - >-
+    latte run --tag latte --duration 50100100 --request-timeout 60 --retry-interval '2s,10s'
+    --sampling 5s --threads 30 --connections 3 --concurrency 180 --rate 25000 -P offset=0
```

Comment:
@vponomaryov 

how did you got to the numbers in threads/connections/concurrency, is that something you calculated, or via experimentations ?  

if the latter, I think it would be good to document that kind of process, not sure where yet...

Reply:
Number of connections is based on the scylla docs saying we should have 2-3 connections.
Threads and concurrency tried to have proportional with some relation to the applied command rate.

I didn't measure perf changes based on the changes to these values.
It is significant amount of time.

Reply:
Can you point which docs you are referring regarding the number of connections ?

Can you elaborate a bit more about what you mean It's proportional to the command ? 

I know experimenting with those values is time consuming, that's exactly why if we have some rules of thumb regarding them, I want them clear and documented as much as possible.

Reply:
> Can you point which docs you are referring regarding the number of connections ?

https://forum.scylladb.com/t/how-many-connections-should-i-open-from-each-scylladb-client-application/43

> Can you elaborate a bit more about what you mean It's proportional to the command ?

`rate` of a command directly influences the need to widen a connections/threads bottleneck.

> I know experimenting with those values is time consuming, that's exactly why if we have some rules of thumb regarding them, I want them clear and documented as much as possible.

It is not something specific to latte. Number of connections, threads and op/s is common among different stress tools.